libp2p/src/builder/phase/quic.rs

```diff
@@ -162,7 +162,6 @@ impl<T: AuthenticatedMultiplexedTransport> SwarmBuilder<super::provider::AsyncSt
         self.without_quic()
```

Comment:
this function (`with_dns` from `quic.rs`) probably doesn't need to remain `async` either